templates/book-section.html

```diff
@@ -3,6 +3,10 @@
 
 {% block head_extensions %}
 <script src="/optional-helpers.js"></script>
+<!-- When this javascript is loaded and initialized it will look for divs with the class `mermaid`
+it will replace the content of those divs with the generated mermaid content-->
+<script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"></script>
```

Comment:
We shouldn't be pulling from a cdn here

Reply:
Do you have a suggestion for a better solution?

Reply:
Changed to use a local version of mermaid instead of fetching from cdn